pom.xml

```diff
@@ -112,6 +128,25 @@ under the License.
 					<target>${java.version}</target>
 				</configuration>
 			</plugin>
+
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
```

Comment:
why do we need to create a jar for these exmaples?

Reply:
This demo is intended to be submitted to a user cluster for testing.